components/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/SAMLSSOUIUtil.java

```diff
@@ -136,6 +137,42 @@ public static boolean isSingleLogoutEnabled(boolean isSpEdit, SAMLSSOServiceProv
         return false;
     }
 
+    /**
+     * Check front-Channel logout enable and if not enable return false.
+     * @param isSpEdit Operation on service provider, create or edit.
```

Comment:
Put a space after the description line, to make it clean to read

Reply:
Fixed with 1149d9757d9ebea69c35422140a80212a346c963